playbook/app/pb_kits/playbook/utilities/test-utils.js

```diff
@@ -39,3 +39,15 @@ export const ensureAccessible = async (Kit, props = {}, newProps = {}) => {
   const html = render()
   expect(await axe(html)).toHaveNoViolations()
 }
+
+// export const appendAlert = (message) => {
```

Comment:
remove or add context 👍

Reply:
Leaving this until I share at dailies and finalize this helper method.